### ISSUE TITLE
Migrating to spring boot 3

### ISSUE THREAD

**Issue by hydraxman** [microsoft/HydraLab#240]
https://spring.io/blog/2022/11/24/spring-boot-3-0-goes-ga
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Release-Notes

**Comment by imnk97** [comment]
Hey Can I work on this issue. Is someone working on this issue?

**Comment by hydraxman** [comment]
Sure, no one works on it yet. And you can try Kotlin if you like it. @imnk97

Here is the [Contribution Guideline](https://github.com/microsoft/HydraLab/blob/main/CONTRIBUTING.md).

**Comment by hydraxman** [comment]
Assigned this to you, feel free to fork a branch and start working on it @imnk97.

**Comment by imnk97** [comment]
Yeah Sure @hydraxman . Already cloned and changed the version but getting some errors so fixing it.

**Comment by hydraxman** [comment]
Thank you. Let me know if we can help in any area.

**Comment by MarianConstantinMarica** [comment]
Hi, it's this one still available, and if yes can I work on it?
